extensions/wikia/CommunityHeader/Navigation.class.php

```diff
@@ -2,93 +2,63 @@
 
 namespace Wikia\CommunityHeader;
 
-use \NavigationModel;
-use \Title;
+use DesignSystemCommunityHeaderModel;
+use NavigationModel;
 
 class Navigation {
 	public $discussLink;
 	public $exploreItems;
 	public $exploreLabel;
 	public $localNavigation;
 
-	public function __construct( $wikiText = null ) {
-		$navigationModel = new NavigationModel();
-		if ( empty( $wikiText ) ) {
-			$this->localNavigation =
-				$navigationModel->getLocalNavigationTree( NavigationModel::WIKI_LOCAL_MESSAGE );
-		} else {
-			$this->localNavigation = $navigationModel->getTreeFromText( $wikiText );
-		}
+	private $model;
+
+	public function __construct( DesignSystemCommunityHeaderModel $model, $wikiText = null ) {
+		$this->model = $model;
+
+		//$navigationModel = new NavigationModel();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this commented out code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS125)